src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataFilesToEcho.java

```diff
@@ -93,7 +93,7 @@ public void readConfiguration(String file) throws IOException, ParseException {
 	 * @param rangeIs360  true if dataset granule coordinates are 0 to 360, null/false if not
 	 * @param boundingBox JSONObject with latMax, lonMax, latMin, lonMin
 	 */
-	public void setDatasetValues(String shortName, String version, Boolean rangeIs360, JSONObject boundingBox) {
+	public void setDatasetValues(String shortName, String version, Boolean rangeIs360, JSONObject boundingBox, JSONObject additionalAttributes) {
```

Comment:
It was an error on my side at he beginning, since when i first worked on the code i was only focused in making a unit test work, which only involved the `.cfg` file; the code has the function in an awkward position such that for CLI, it uses `readConfiguration` only, while for Lambda it uses `setDatasetValues` directly as is